qcengine/util.py

```diff
@@ -146,13 +146,13 @@ def handle_output_metadata(
     metadata: Dict[str, Any],
     raise_error: bool = False,
     return_dict: bool = True,
-) -> Union[Dict[str, Any], "BaseModel"]:
+) -> Union[Dict[str, Any], "AtomicResult", "OptimizationResult", "FailedOperation"]:
```

Comment:
I wonder if it might be clearer here to leave the `-> ... "BaseModel"]:` since BaseModel is mentioned in `output_data`. The likely AtRes, OptRes, FailedOp could be a note in the docstring.

Reply:
Will try and finish up tonight!

Reply:
Good point. I think it may be better the other way; i.e., have `output_data` more specifically define the classes that this function will actually return. I agree the discrepancy is confusing!

Reply:
Let me know if you'd like me to go the other way on this. It appears the function is only used in the `compute()` and `compute_procedure()` functions so it should only be returning the values I added the the `ouput_data` annotation.